### PR TITLE
Dependencies Update

### PR DESCRIPTION
**Pull request by prescod**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
         run: |
           python -VV
           python -m pip install --upgrade pip
-          make dev-install
+          pip install -r requirements_dev.txt
```
Comment (boakley) [inline .github/workflows/ci.yml:27]: You removed the use of dev-install but left it in Makefile and it's still referenced in the README and CONTRIBUTING files. Was that all intentional?

**Review by boakley — APPROVED** [review]
I left one small comment, but otherwise it looks good to me.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (prescod) [reply .github/workflows/ci.yml:27]: I'm hoping in a week or so `make dev-install` will work again when pip-tools is fixed.

Comment (jstvz) [reply .github/workflows/ci.yml:27]: > I'm hoping in a week or so `make dev-install` will work again when pip-tools is fixed.


I don't think that's necessary. `make dev-install` is a target is for our convenience, and wasn't doing anything special in our CI builds. (The reason it calls `pip-sync` is to synchronize a virtualenv with the requirements files generated by `pip-compile`, but there's no venv to synchronize in the CI runner).

**Pull request merged**